windows-nsis/openvpn.nsi

```diff
@@ -508,13 +508,12 @@ Function .onInit
 
 ${IfNot} ${AtLeastWinVista}
 
-	MessageBox MB_YESNO|MB_ICONEXCLAMATION "This package does not work on your operating system.  The last version of OpenVPN supported on your OS is 2.3. Shall I open a web browser for you to download it?" /SD IDYES IDNO guiEndNo
-	Goto guiEndYes
+	MessageBox MB_YESNO|MB_ICONEXCLAMATION "This package does not work on your operating system.  The last version of OpenVPN supported on your OS is 2.3. Shall I open a web browser for you to download it?" /SD IDYES DownloadForWinXP IDNO DontDownloadForWinXP
```

Comment:
I see, it  has to be  /SD IDYES IDYES jump_if_IDYES IDNO jump_if_IDNO
@chipitsine 's suggestion in PR61 may be used here (or merge that one first) PR 61 also changes the /SD IDYES to /SD IDNO which is better for silent install.